### PR TITLE
docs: add pull request template & update contributing docs.

### PR DESCRIPTION
**Pull request by qicz**
Fixes: #1315

### CODE REVIEW

File: docs/latest/dev/CONTRIBUTING.md [hunk 1]
```diff
@@ -52,7 +52,9 @@ to the following guidelines for all code, APIs, and documentation:
 * Your PR title should be descriptive, and generally start with a subsystem name followed by a
   colon. Examples:
   * "docs: fix grammar error"
-  * "translator: add new feature"
+  * "feat(translator): add new feature"
+  * "bugs: fix bug"
```
Comment (Xunzhuo) [inline docs/latest/dev/CONTRIBUTING.md:56]: `fix: xxxx` is better than `bugs: xxxx`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+**What type of PR is this?**
+<!--
+Add one of the following kinds:
+/feat: new feature
+/bugs: fix bug
```
Comment (Xunzhuo) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: https://www.conventionalcommits.org/en/v1.0.0/#summary

Comment (Xunzhuo) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: IMO, the `/xxx` style is used in k8s communities and prow style to add label automatically. Like:

/kind bug
/kind feature
/kind refactor

etc.

File: docs/latest/dev/CONTRIBUTING.md (hunk 1 above)
Comment (Xunzhuo) [reply docs/latest/dev/CONTRIBUTING.md:56]:
```
feat: add hat wobble
^--^  ^------------^
|     |
|     +-> Summary in present tense.
|
+-------> Type: chore, docs, feat, fix, refactor, style, or test.

```

**Comment by zirain** [comment]
EG use `squash` merge, do not `push -f` every time for better review.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+**What type of PR is this?**
+<!--
+Your PR title should be descriptive, and generally start with type that contains a subsystem name with `()` if necessary 
+and summary followed by a colon. format `chore/docs/feat/fix/refactor/style/test: summary`.
+Examples:
+* "docs: fix grammar error"
+* "feat(translator): add new feature"
+* "fix: fix xx bug"
```
Comment (zirain) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: it would be better to add a action for auto-label.

**Comment by qicz** [comment]
> EG use `squash` merge, do not `push -f` every time for better review.

I know `squash` will merge all commits into one commit. sometimes `-f` to fix codecov timeout or clarify the commit message.

**Review by zirain — APPROVED** [review]

**Pull request merged**